relay-server/src/metrics_extraction/transactions.rs

```diff
@@ -160,23 +611,33 @@ fn get_transaction_name(
     name
 }
 
+#[derive(Default, Clone)]
+struct UniversalTags {
+    release: Option<String>,
+    dist: Option<String>,
+    environment: Option<String>,
+    transaction: Option<String>,
+    platform: Option<String>,
+    transaction_status: Option<String>,
+    transaction_op: Option<String>,
+    http_method: Option<String>,
+    other: BTreeMap<String, String>,
```

Comment:
```suggestion
    custom_tags: BTreeMap<String, String>,
```

The only source for this `other` category are custom tags, right?

Reply:
yeah, at least thats how i interpret this code from `extract_universal_tags`

```
let custom_tags = &config.extract_custom_tags;
    if !custom_tags.is_empty() {
        // XXX(slow): event tags are a flat array
        if let Some(event_tags) = event.tags.value() {
            for tag_entry in &**event_tags {
                if let Some(entry) = tag_entry.value() {
                    let (key, value) = entry.as_pair();
                    if let (Some(key), Some(value)) = (key.as_str(), value.as_str()) {
                        if custom_tags.contains(key) {
                            tags.other.insert(key.to_owned(), value.to_owned());
                        }
                    }
                }
            }
        }
    }
    ```